apple/RNGestureHandler.m

```diff
@@ -176,6 +176,38 @@ - (void)setEnabled:(BOOL)enabled
   self.recognizer.enabled = enabled;
 }
 
+#if !TARGET_OS_OSX
+- (void)setCurrentPointerType:(UIEvent *)event
+{
+  UITouch *touch = [[event allTouches] anyObject];
+
+  switch (touch.type) {
+    case UITouchTypeDirect:
+      _pointerType = RNGestureHandlerTouch;
```

Comment:
There is a small difference between naming on Android and iOS. On Android we call this type of pointer `POINTER_TYPE_FINGER` and here it's called `RNGestureHandlerTouch`. Do we want to rename to something like `POINTER_TYPE_TOUCH` or `RNGestureHandlerFinder`?

Reply:
I think it would make sense to stick to `touch`.

Reply:
Android done in [025f549](https://github.com/software-mansion/react-native-gesture-handler/pull/2760/commits/025f5491ece30c8cae814d737d8589c466eb61e8).
Web done in [f74ce39](https://github.com/software-mansion/react-native-gesture-handler/pull/2760/commits/f74ce39c515e028d889eee9f27a1de348364624a).